app/models/appointment_summary.rb

```diff
@@ -4,8 +4,9 @@
 
 class AppointmentSummary < ActiveRecord::Base
   DIGITAL_BY_DEFAULT_START_DATE = Date.new(2016, 6, 21)
+  # bassed off: https://github.com/alphagov/notifications-utils/blob/master/notifications_utils/recipients.py#L22
   EMAIL_REGEXP = Regexp.union(
-    /\A[^\@\s]+@[^\@\.\s]+(\.[^\@\.\s]+)+$/,
+    /\A([^";@\s\.]+\.)*[^";@\s\.]+@([^";@\s\.]+\.)+[a-z]{2,10}\z/,
```

Comment:
Couldn't we just switch to using an email field here:

https://github.com/guidance-guarantee-programme/output/blob/master/app/views/appointment_summaries/new.html.erb#L39

I'm not really a fan of whitelisting email formats as we go along, we'll potentially lose customers?